backend/functions/utils.py

```diff
@@ -185,17 +193,25 @@ def get_translation_using_cdac_model(input_sentence, source_language, target_lan
     return response.json()["output"][0]["target"]
 
 
-def get_batch_translations_using_google_translate(sentence_list, target_language):
+def get_batch_translations_using_google_translate(sentence_list, target_language, checks_for_particular_languages=False):
     """Function to get the translation for the input sentences using the Google Translate API.
 
     Args:
         sentence_list (str): List of sentences to be translated.
         target_language (str): Final language of the sentence.
+        checks_for_particular_languages (bool): If True, checks for the particular languages in the translations.
 
     Returns:
         list: List of dictionaries containing the translated sentences.
     """
 
+    if checks_for_particular_languages: 
+        # Checks for particular languages
+        if target_language in ["Bodo", "Maithili"]: 
+            target_language = "Hindi"

```

Comment:
I think Maithili is already supported in Google.
Can you please check once.

Reply:
@ishvindersethi22 requested this change. Can you please add your thoughts?

Reply:
But the annotators need it in Hindi as per requirements.